### PR TITLE
Fixed empty line to

### LINKED ISSUE #43: Hex::line_to always returns the origin when start==end
Specialcasing this in my code fixed the bug I was seeing, so it's just a simple bad computation :)

### PR DESCRIPTION
**Pull request by ManevilleF**
> Closes #43 

Fixes empty `line_to` implementation. With this fix the returned line will always contain at least both `start` and `end` coordinates.

### CODE REVIEW

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
~~Can we add a test specifically for the `start == end` case when it's non-zero?~~

Oh I see, that's what the test actually does. Too early here!

File: src/hex/tests.rs [hunk 1]
```diff
@@ -337,6 +337,13 @@ fn line_to() {
     );
 }
 
+#[test]
+fn empty_line_to() {
+    let start = Hex::new(3, -7);
+    let line: Vec<_> = start.line_to(start).collect();
+    assert_eq!(line, vec![start, start]);
```
Comment (alice-i-cecile) [inline src/hex/tests.rs:344]: This seems like the wrong result still: I would expect a vec of length one, with only `start` as the element.

**Review by alice-i-cecile — APPROVED** [review]
